seed/views/v3/tax_lot_properties.py

```diff
@@ -112,25 +111,23 @@ def export(self, request):
             profile_id,
             view_klass_str)
         column_name_mappings.update(add_column_name_mappings)
+
         select_related = ['state', 'cycle']
+        prefetch_related = ['labels']
         ids = request.data.get('ids', [])
         filter_str = {'cycle': cycle_pk}
+        if ids:
+            filter_str['id__in'] = ids
         if hasattr(view_klass, 'property'):
             select_related.append('property')
-            prefetch_related = ['labels']
-            filter_str = {'property__organization_id': org_id}
-            if ids:
-                filter_str['id__in'] = ids
+            filter_str['property__organization_id'] = org_id
             # always export the labels and notes
             column_name_mappings['property_notes'] = 'Property Notes'
             column_name_mappings['property_labels'] = 'Property Labels'
 
         elif hasattr(view_klass, 'taxlot'):
             select_related.append('taxlot')
-            prefetch_related = ['labels']
-            filter_str = {'taxlot__organization_id': org_id}
-            if ids:
-                filter_str['id__in'] = ids
+            filter_str['taxlot__organization_id'] = org_id
```

Comment:
a little code cleanup. Note that originally we were dropping the `cycle` filter param every time (we were always resetting `filter_str`), this fixes that as well.